qrisk/stats.py

```diff
@@ -353,15 +431,22 @@ def sortino_ratio(returns, required_return=0, period=DAILY):
         Annualized Sortino ratio.
 
     """
-    try:
-        ann_factor = ANNUALIZATION_FACTORS[period]
-    except KeyError:
-        raise ValueError(
-            "period cannot be: '{}'."
-            " Must be '{}', '{}', or '{}'".format(
-                period, DAILY, WEEKLY, MONTHLY
+
+    if len(returns) < 2:
+        return np.nan
+
+    if annualization is None:
```

Comment:
There are some years that don't have 252 business days in them. 2017 is one of them.